en_ewt-ud-train.conllu

```diff
@@ -246794,7 +246794,7 @@
 27	that	that	PRON	WDT	PronType=Rel	29	obj	26:ref	_
 28	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	29	nsubj	29:nsubj	_
 29	loved	love	VERB	VBD	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin	26	acl:relcl	26:acl:relcl	_
-30	compared	compare	VERB	VBN	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	20	obl	20:obl:about	_
+30	compared	compare	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	20	obl	20:obl:about	_
```

Comment:
I think VBN is correct. Should be VerbForm=Part

Reply:
Fixed, thanks.

Reply:
wait never mind